CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased: mitmproxy next
 
 
+Enhance documentation and add alert log messages when stream_large_bodies and modify_body are set (#6479)
```

Comment:
```suggestion
* Enhance documentation and add alert log messages when stream_large_bodies and modify_body are set
```

CI will do the linking, but it needs to be a bullet point for that. :)

Reply:
Thanks for that tip.